scripts/dumps/prepare_items_dumps

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+source ./scripts/dumps/lib/utils.sh
+
+items_dump="${today_folder}/items.ttl"
+dbHost=$(node -p "require('config').db.fullHost()")
+
+echo "
+@prefix wdt: <http://www.wikidata.org/prop/direct/> .
+@prefix inv: <http://inventaire.io/entity/> .
+@prefix invexamplar: <http://inventaire.io/items/> .
```

Comment:
wuhu what a brand new name ha ?!

Reply:
maybe `invex` as its should stay a short string as a prefix ?

Reply:
yeah that was temporary, and with the whole items dump being commented-out now, that could even be removed

Reply:
renamed in c556bb8e